azure/src/App.js

```diff
@@ -1,15 +1,60 @@
 import Navbar from "./components/Navbar";
 import Banner from "./components/Banner";
 import Card from "./components/Card";
-
+import CardData from "./components/CardData";
+import Cardimg1 from "./assets/CardImg.webp";
+import Imagea from "./assets/Imagea.webp";
+import LinuxImg from "./assets/LinuxImg.webp";
+import Section from "./components/Section";
+import Linux from "../src/assets/Linux.svg";
+import AzureServiceCard from "./components/AzureServiceCard";
+import Explore from "./components/Explore";
+import Products from "./components/Products";
+import React,{useState} from "react";
+import AzureInnovation from "./components/AzureInnovation";
+import AzureMigration from "./components/AzureMigration";
+import DiscoverCards from "./components/DiscoverCards";
```

Comment:
Lets always follow this order for imports:
`import React from 'react';

import { PropTypes } from 'prop-types';
import styled from 'styled-components/native';

import ErrorImg from '../../assets/images/error.png';
import colors from '../../styles/colors';`

The closest resource should be imported last. Also try to give a blank line after external imports and after similar imports like components etc